xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-livetable/src/main/java/org/xwiki/livedata/internal/livetable/LiveTableLiveDataConfigurationResolver.java

```diff
@@ -323,7 +323,8 @@ private DisplayerDescriptor getDisplayerConfig(String column, ObjectNode columnP
         DisplayerDescriptor displayerConfig = new DisplayerDescriptor();
         if (columnProperties.path(ACTIONS).isArray()) {
             displayerConfig.setId(ACTIONS);
-            displayerConfig.setParameter(ACTIONS, columnProperties.get(ACTIONS));
+            displayerConfig.setParameter(ACTIONS,
+                columnProperties.get(ACTIONS).valueStream().map(JsonNode::asText).toList());
```

Comment:
Pointing out that I had to replace arrays with list to values comparable with what is in the test.